pyreisejl/utility/extract_data.py

```diff
@@ -246,10 +238,11 @@ def _get_outputs_from_converted(matfile):
     }
 
     try:
+        # If DC lines are present in the input file, use their indices
         outputs_id["pf_dcline"] = case.mpc.dclineid
+        outputs_id["trans_viol"] = list(case.mpc.branchid) + list(case.mpc.dclineid)
```

Comment:
I was thinking whether we should separate ac lines and dc lines in terms of capacity violation?

Reply:
The pre-existing functionality has the `trans_viol` variable defined over all branches, AC & DC. We could limit it to AC branches only if we want, I just added extraction to match what's already here.